fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordCursorResult.java

```diff
@@ -200,6 +200,14 @@ public int hashCode() {
         }
     }
 
+    /**
+     * Returns true if the cursor has reached its end but a continuation is still available (i.e. the source is not yet exhausted).
+     * @return {@code true} if this result does not include a next value but include a continuation and {@code false} otherwise
+     */
+    public boolean noNextButHasContinuation() {
```

Comment:
This method seems very useful but somehow the name seems a bit confusing. How do you feel about `isStoppedWithContinuation()` or `hasStoppedBeforeEnd()` or something along those lines? This is a key part of our external API so I thin it's worth getting it right (so it can be stable).

Reply:
Yeah, I like `hasStoppedBeforeEnd()`